### ISSUE TITLE
Pretrained model

### ISSUE THREAD

**Issue by Jimmy880** [usc-sail/media-eval-2020#1]
Thanks for your great work. It is really helpful for further study.
Could you please provide pre-trained model that won the media-eval-2020 competition? I would like to use the pre-trained model for auto-tagging music mood

**Comment by timothydgreer** [comment]
Hi Jimmy,
Thanks for your kind words. We are retraining our model now and should have the winning model up soon. We'll respond again when it is up! In the meantime, you can use the MSD model, which is in the github.
If you're in a time crunch, you should be able to follow the readme file to create the model yourself. And do ask if you have any questions, clarifications, or issues!

**Comment by timothydgreer** [comment]
Hi Jimmy,
The best performing individual model (focal loss) is now up on the github! It is in media-eval-2020/pretrained_model/focal_loss_all_data/
I'm going to close this issue, as I think this should answer your question! Let me know if you run into any other issues.

**Comment by Jimmy880** [comment]
Thanks for your kindly help!! I will follow the guideline in the readme file and check the model carefully.
Thank you again, It is really helpful.
